packages/manager/modules/hub/src/components/carousel/carousel.controller.js

```diff
@@ -0,0 +1,57 @@
+import map from 'lodash/map';
+import first from 'lodash/first';
+import set from 'lodash/set';
+import find from 'lodash/find';
+import findIndex from 'lodash/findIndex';
+import filter from 'lodash/filter';
+
+import { Environment } from '@ovh-ux/manager-config';
+
+export default class {
+  /* @ngInject */
+  constructor($window, OvhApiNotificationAapi, TranslateService) {
+    this.$window = $window;
+    this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+    this.TranslateService = TranslateService;
+    this.REGION = Environment.getRegion();
+  }
+
+  $onInit() {
+    return this.OvhApiNotificationAapi.query(
+      this.TranslateService.getUserLocale(),
```

Comment:
```$translate.use()```?

Reply:
I actually think it is better using `TranslateService` because it abstracts a little the way we get user's locale